### DISCUSSION TITLE
Function key under egui::Key

### DISCUSSION THREAD

**Discussion by resonant-jovian** [emilk/egui#795] [Ideas]
Hey!

I am making a game with egui for ui and I would really like to use the Function keys for ui if statements.
Is this something you could consider implementing?

/Albin

**Comment by emilk** [comment]
I'm don't think it is possible for an application to read the state of the function key. [winit doesn't expose it](https://docs.rs/winit/0.25.0/winit/event/enum.VirtualKeyCode.html).

**Reply by resonant-jovian** [reply]
Okay I might have to write some code for that myself then.

**Reply by AlexApps99** [reply]
I don't recommend using egui for handling game input, unless it's an absolutely integral part of your game.
